esx_service/cli/vmdkops_admin.py

```diff
@@ -1301,16 +1305,38 @@ def config_rm(args):
     # This asks for double confirmation, and removes the local link or DB (if any)
     # NEVER deletes the shared database - instead prints help
 
-    if not args.local:
+    if not args.local and not args.unlink:
         return err_out("""
         Shared DB removal is not supported. For removing  local configuration, use --local flag.
-        For removing shared DB,  run 'vmdkops_admin config rm --local' on ESX hosts using this DB,
+        For removing shared DB,  run 'vmdkops_admin config rm --unlink' on ESX hosts using this DB,
         and manually remove the {} file from shared storage.
         """.format(auth_data.CONFIG_DB_NAME))
 
+    # Check the existing config mode
```

Comment:
Missing check 
`if args.local and args.unlink: err_out("cannot use both flags together" + help message above  about when to use what)